hummingbot/connector/exchange_py_base.py

```diff
@@ -259,6 +260,7 @@ def quantize_order_amount(self, trading_pair: str, amount: Decimal, price: Decim
 
         # Add 1% as a safety factor in case the prices changed while making the order.
         if notional_size < trading_rule.min_notional_size * Decimal("1.01"):
+            self.logger().warning("Quantizing order amount to 0 because order notional value is not up to expected minimum notional value.")
```

Comment:
Please replace with:
`self.logger().warning(f"Quantizing order amount to 0 because order notional value is below {trading_rule.min_notional_size} market minimum notional value.")`